### PR TITLE
fix mail to link handling

### LINKED ISSUE #11: Misconfigured mailto link
On the first page the link to the google group is 
"mailto:https://groups.google.com/forum/#!forum/gonum-dev"
causing it to open my email client instead of linking to the page. 

This is probably caused by the email being configured to be that link and the page being set up to auto generate link.

### PR DESCRIPTION
**Pull request by kortschak**
I'm waiting on one issue from gcushen, so this is not ready to merge.

Fixes #11.

### CODE REVIEW

File: content/home/posts.md [hunk 1]
```diff
@@ -12,8 +12,11 @@ widget = "posts"
 # Order that this section will appear in.
 weight = 2
 
-# Show posts that contain the following tags. Default to any tags.
-tags = []
+# Filter posts by tag.
+#  By default, show all recent posts.
+#  Filtering example: `tags_include = ["hugo", "academic"]`
+tags_include = []
```
Comment (btracey) [inline content/home/posts.md:18]: Does this mean "dont' filter anything"?

File: content/post/intro-to-stats-with-gonum.md [hunk 2]
```diff
@@ -1,8 +1,18 @@
----
-title: "Introduction to Statistics With Gonum"
-date: 2017-10-04T11:17:11+02:00
-Categories: ["go", "gonum", "stat"]
----
++++
+date = "2017-10-04T11:17:11+02:00"
+draft = false
+tags = ["stat"]
```
Comment (btracey) [inline content/post/intro-to-stats-with-gonum.md:4]: The assumption being the other tags are redundant?

File: content/home/posts.md (hunk 1 above)
Comment (kortschak) [reply content/home/posts.md:18]: I think so.

File: content/post/intro-to-stats-with-gonum.md (hunk 2 above)
Comment (kortschak) [reply content/post/intro-to-stats-with-gonum.md:4]: Yes.

**Review by sbinet — APPROVED** [review]

**Comment by kortschak** [comment]
I have the previous problem fixed, but it introduces another. I don't have enough time to dig into the css to fix this, so if someone else has any suggestions, please post it here.

**Comment by kortschak** [comment]
PTAL

**Comment by kortschak** [comment]
I've tried adding a CSS element into the override.css file to get the "GONUM" to be less shouty, but it doesn't work for me, I copied the stanza in academic.css

```css
.navbar-default .navbar-brand {
  text-transform: uppercase;
  font-weight: bold;
  font-size: 1.2em;
  color: {{ .Get "menu_title" }};
}
```

to

```css
.navbar-default .navbar-brand {
  font-weight: bold;
  font-size: 1.2em;
  color: {{ .Get "menu_title" }};
}
```

If someone else understand the dark arts of CSS and can fix that, that would be good.

**Comment by sbinet** [comment]
ping?

**Comment by kortschak** [comment]
I don't know how to deal with this. It's been too long. I'm going to close this and we can bring the version of hugo being used up-to-date and then have another go.

**Pull request closed without merging**